coverage/clover.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What was the reason you decided to push the coverage folder?

Reply:
There is no major reason. I just had one thought to let reviewer quickly go over the html report(./coverage/lcov-report/index.html)instantly that was generated before my push.